src/main/java/app/model/Map.java

```diff
@@ -37,7 +37,7 @@
 
     public Map(Settings settings)
     {
-        System.out.print("Loading settings ... ");
+        System.out.println("Loading settings ... ");
```

Comment:
Why did you do this?   The whole point of this (and the last line) is that it makes a nice single line output
```
Loading settings ... done
```

Why are you even messing with this?
What did my code even do to you?!

Reply:
I added print-statements in case that the number of guards doesn't fit into the spawn area, if those are printed, it get's put between them
So it seemed best to make it a seperate line

Reply:
but if gonna make a runtime error instead, it isn't necessary, so will change back